blocks/init/src/Blocks/components/button/button.php

```diff
@@ -42,53 +43,45 @@
 	);
 }
 
-$buttonWrapClass = Components::classnames([
-	Components::selector($componentClass, "{$componentClass}-wrap"),
-	Components::selector($buttonAlign, "{$componentClass}-wrap", 'align', $buttonAlign),
-	Components::selector($blockClass, $blockClass, "{$selectorClass}-wrap"),
-]);
-
 $buttonClass = Components::classnames([
 	$componentClass,
-	Components::selector($buttonColor, $componentClass, 'color', $buttonColor),
-	Components::selector($buttonSize, $componentClass, 'size', $buttonSize),
-	Components::selector($buttonWidth, $componentClass, 'size-width', $buttonWidth),
+	Components::selector($buttonAsLink, $componentClass, 'as-link'),
```

Comment:
Would `is-link` be a better action modifier?

Reply:
updated